packages/block-library/src/query/edit/query-placeholder.js

```diff
@@ -79,8 +79,7 @@ export default function QueryPlaceholder( {
 			>
 				{ !! hasPatterns && (
 					<Button
-						// TODO: Switch to `true` (40px size) if possible
-						__next40pxDefaultSize={ false }
+						__next40pxDefaultSize
```

Comment:
This can be seen while you add the `core/query` block and check for the button `Choose`.

| Before | After     |
| :-------- | :------- |
| <img width="772" alt="placeholder-before" src="https://github.com/user-attachments/assets/b7d07d2c-6174-4c93-adcc-2b55c2cd035a"> | <img width="772" alt="placeholder-after" src="https://github.com/user-attachments/assets/2092b64e-bba0-4c97-8114-f3f66e196345"> |